projects/freetype2/Dockerfile

```diff
@@ -21,3 +21,10 @@ RUN apt-get install -y make autoconf libtool libarchive-dev
 RUN git clone --depth 1 git://git.sv.nongnu.org/freetype/freetype2.git
 WORKDIR freetype2
 COPY build.sh $SRC/
+
+# Get some files for the seed corpus
+RUN wget https://github.com/adobe-fonts/adobe-variable-font-prototype/releases/download/1.001/AdobeVFPrototype.otf
+
+
+# Put all seed files into $OUT/ftfuzzer_seed_corpus.zip
+RUN zip -j $OUT/ftfuzzer_seed_corpus.zip $(find . -name '*.otf')
```

Comment:
Since `$SRC` works here, I'd expect `$OUT` to work as well, but if I do `shell` after `build_image`, I don't see the archive in `/out`. Looks like `/out` becomes flushed or re-mounted somehow. Just to test, I've added `RUN cp /out/ftfuzzer_seed_corpus.zip /src/` to the end of the Dockerfile, then I've seen the archive in `/src/`, i.e. it has been in `/out/` at some point.

Anyway, I think it makes sense to create seed corpus always in `build.sh` after building the fuzzer. Otherwise it can be accidentally overwritten and partially lost (i.e. somebody adds seed corpus in `build.sh` later and misses this line in Dockerfile). I suggest to move line 30 to the end of `build.sh`.